KubeArmor/BPF/shared.h

```diff
@@ -174,7 +174,8 @@ static __always_inline bool prepend_path(struct path *path, bufs_t *string_p) {
       m = BPF_CORE_READ(mnt, mnt_parent);
       if (mnt != m) {
```

Comment:
```suggestion
      mnt_parent = BPF_CORE_READ(mnt, mnt_parent);
      if (mnt != mnt_parent) {
```

QoL update, can we update the m variable to mnt_parent to better understand what's happening